test/lib/usd/translators/testUsdExportCustomRenderContext.py

```diff
@@ -0,0 +1,83 @@
+#!/pxrpythonsubst
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pxr import Plug
+from pxr import Usd
+from pxr import UsdShade
+
+import os
+import unittest
+
+from maya import cmds
+from maya import standalone
+
+import fixturesUtils
+
+class testUsdExportCustomRenderContext(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.input_path = fixturesUtils.setUpClass(__file__)
+
+        test_dir = os.path.join(cls.input_path,
+                                "UsdExportCustomRenderContextTest")
+
+        cmds.workspace(test_dir, o=True)
+
+    @classmethod
+    def tearDownClass(cls):
+        standalone.uninitialize()
+
+    def testUsdExportCustomRenderContext(self):
+        """
+        Tests a custom exporter for a render context that exists in an unloaded
+        plugin.
+        """
+        plugin_path = os.path.join(self.input_path, "..", "plugin")
+        Plug.Registry().RegisterPlugins(plugin_path)
```

Comment:
Can we do this by setting `PXR_PLUGINPATH_NAME` in the build system rather than having the test assume it knows how files are organized above it?